### PR TITLE
Assignment 1

### PR DESCRIPTION
**Pull request by DanDanBro**

### CODE REVIEW

**Review by MattiaBonfanti-CS — APPROVED** [review]
Looks good

**Review by LaurensKrudde — APPROVED** [review]
Awesome!

File: README.md [hunk 1]
```diff
@@ -1,2 +1,42 @@
...
+# Add GitHub Personal Access Token to environment (linux)
+export NPM_TOKEN=the_token_goes_here
+
+# Add GitHub Personal Access Token to environment (windows)
+set NPM_TOKEN=the_token_goes_here
+```
+
+```
+# Docker build
+docker build -t ghcr.io/remla23-team15/app:VERSION --build-arg npm_token=${NPM_TOKEN} .
```
Comment (MattiaBonfanti-CS) [inline README.md]: Should we also add the command for windows?

docker build -t ghcr.io/remla23-team15/app:VERSION --build-arg npm_token=%NPM_TOKEN% .

Comment (LaurensKrudde) [reply README.md]: True! I forgot that one. I'll change it

**Pull request merged**